add_npm_package.sh

```diff
@@ -138,7 +139,7 @@ if [[ -z $STRATEGY ]] ; then
   echo "Found $DEPENDENCIES dependencies - using $STRATEGY strategy"
 fi
 
-if [ -f "$PACKAGE_DIR"/*.spec ]; then
```

Comment:
Context: I think this predates from when we properly aligned all spec files to match their directory name. These days we can rely on that.